docs/hooks/use-view-transition-state.md

```diff
@@ -30,4 +30,4 @@ function NavImage({ src, alt, id }) {
 [view-transitions]: https://developer.mozilla.org/en-US/docs/Web/API/View_Transitions_API
 [link-component-view-transition]: ../components/link#unstableviewtransition
 [form-component-view-transition]: ../components/form#unstableviewtransition
-[nav-linnk-component-view-transition]: ../components/nav-link#unstableviewtransition
+[nav-link-component-view-transition]: ../components/nav-link#unstableviewtransition
```

Comment:
I think this link is actually wrong. Same with `link-component-view-transition` and `form-component-view-transition`

```suggestion
[nav-link-component-view-transition]: ../components/nav-link#unstable_viewtransition
```